radio/src/targets/horus/board.cpp

```diff
@@ -89,11 +87,19 @@ void interrupt1ms()
   DEBUG_TIMER_START(debugTimerRotEnc);
   checkRotaryEncoder();
   DEBUG_TIMER_STOP(debugTimerRotEnc);
+
+#if defined(LUA)
+  if (telemetryProtocol == PROTOCOL_FRSKY_SPORT) {
+    if (telemetryFifo.last(-2) == 0x7E && telemetryFifo.last(-1) == outputTelemetryBufferTrigger) {
+      sportSendBuffer(outputTelemetryBuffer, outputTelemetryBufferSize);
```

Comment:
Just to precise, it does NOT seem to be related to "normal" telemetry losses, I can have as many of those as I want without issues, the "crash" where you get "telemetry lost" and it won't recover without a reboot of the radio can happen at any distance (reproduced it in range check mode 10m away too).